bids_prov/spm_load_config.py

```diff
@@ -23,12 +23,12 @@
 def get_empty_graph(context_url=CONTEXT_URL):
     return {
         "@context": context_url,
-        "@id": "http://example.org/ds00000X",
-        "generatedAt": "2020-03-10T10:00:00",
+        "BIDSProvVersion": "1.0.0",  # TODO ?
+        "@id": "http://example.org/ds00000X",  # TODO ?
         "wasGeneratedBy": {
             "@id": "INRIA",
             "@type": "Project",
-            "startedAt": "2016-09-01T10:00:00",
+            "startedAt": "2016-09-01T10:00:00",  # TODO ?
```

Comment:
Yes to remove as we don't have this info in SPM's matlabbatch

Reply:
erased